### PR TITLE
Card Generator: Added Font-Family selector with +10 fonts

### PR DESCRIPTION
**Pull request by StalkerVince**

### CODE REVIEW

**Review by Chikoulat — APPROVED** [review]

**Review by Lambseb — APPROVED** [review]
line 76  text not necessary  " >
          Bonjour, je suis un texte de test! XD"
  good job however

File: frontend/src/components/CardGeneration.jsx [hunk 1]
```diff
@@ -15,8 +25,65 @@ function CardGeneration() {
         type="color"
         value={savedFontColor}
       />
+      <select>
```
Comment (ioayoub) [inline frontend/src/components/CardGeneration.jsx]: Well, fix this please :)

**Review by ioayoub — CHANGES_REQUESTED** [review]

**Review by ioayoub — APPROVED** [review]
Top !

**Review by ioayoub — CHANGES_REQUESTED** [review]
Conflicts

**Pull request closed without merging**